source/_integrations/systemmonitor.markdown

```diff
@@ -64,7 +64,7 @@ file.
 | packets_out            | Interface, e.g., `eth0`   | yes                       |
 | ipv4_address           | Interface, e.g., `eth0`   | yes                       |
 | ipv6_address           | Interface, e.g., `eth0`   | yes                       |
-| processor_use          |                           |                           |
```

Comment:
The type is `processor_use` and not `processor_use_percent`.

https://github.com/home-assistant/core/blob/dev/homeassistant/components/systemmonitor/sensor.py#L82

Reply:
https://github.com/home-assistant/core/pull/41411 was changing the entity ID but that's different than the configuration value.